{"uuid": "318cbce3-1fd0-48f9-ae36-b8d032a4b993", "children": ["762ea09e-fbe1-4412-8e36-dab2679c5887"], "befores": [{"name": "pw_encryption", "status": "broken", "statusDetails": {"message": "TypeError: 'SubRequest' object is not iterable\n", "trace": "  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Administrator\\PycharmProjects\\test\\conftest.py\", line 23, in pw_encryption\n    pw = list(request)\n"}, "start": 1668157855661, "stop": 1668157855661}], "start": 1668157855661, "stop": 1668157855795}